data_pipeline/scraper/util.py

```diff
@@ -0,0 +1,32 @@
+import os
+from os import path
+import logging
+import time
+from random import randint
+
+logger = logging.getLogger(__name__)
+
+
+def renameDownloadedFile(absolutefname: str):
+    '''
```

Comment:
If the file does not get overridden does that mean we will need to go in and manually clean those files up every once in a while or a script for it?

Reply:
This is correct & consistent with the current behavior on windows & mac. Usually can just delete the whole data directory to start clean.  We may want to improve this in the future (perhaps delete this directory whenever we start the script?) but this is lower priority.